### PR TITLE
fix #588 Refactor review to revision 

### LINKED ISSUE #588: Review should be updated to Revision
As per new mocks, Topic Review Tab has been updated to Topic Revision Tab,

Screenshot 2020-01-03 at 12 11 29 PM

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Review` with `Revision`.

### PR DESCRIPTION
**Pull request by maheshpaliwal**
## Explanation

Fixes #588 
all files related to review has been updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by maheshpaliwal** [comment]
@rt4914 PTAL

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml:165]: Please remove this file from your commit history. These changes must have come from android studio automatically.

File: app/src/sharedTest/java/org/oppia/app/topic/revision/TopicReviewFragmentTest.kt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.oppia.app.topic.review
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/topic/revision/TopicReviewFragmentTest.kt]: @maheshpaliwal This file name is still `TopicReviewFragmentTest`, please change that to `TopicRevisionFragmentTest`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Good work @maheshpaliwal . I have mentioned couple of changes. PTAL

**Comment by rt4914** [comment]
@maheshpaliwal also your PR title should be:
`Fix #588: Refactor review to revision`

**Comment by maheshpaliwal** [comment]
@rt4914 worked on suggested changes PTAL

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (rt4914) [reply .idea/codeStyles/Project.xml:165]: This file is still here, we need to revert this entire file.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@maheshpaliwal There is force push on this branch. Can you please explain why? Because in oppia we do not accept PR which have pull request unless it is entirely necessary.

**Comment by maheshpaliwal** [comment]
I did force push because I wanted to edit commit history. Since force push is not accepted i'm closing this pull request and starting new one

**Comment by rt4914** [comment]
> I did force push because I wanted to edit commit history. Since force push is not accepted i'm closing this pull request and starting new one

Sounds good. No issues. We normally do not alter the commit history. Instead the next commit fixes  those changes. Thanks :-)

**Comment by rt4914** [comment]
Closing this as there has been no update on this for past 1 month and the contributor has been lapsed.

**Pull request closed without merging**